### PR TITLE
updates for razor pages crud and blazor crud templates

### PR DESCRIPTION
**Pull request by deepchoudhery**
addressing missing content after fixing https://github.com/dotnet/Scaffolding/issues/2899 and https://github.com/dotnet/Scaffolding/issues/2891
Blazor-CRUD changes : 
- adding a `*` alongside the label for required properties in the Create and Edit razor files (similar to Razor pages now)
- above change fixes the need for a visual indicator before a submit operation.
Razor pages CRUD changes : 
- adding `required aria-required="true"` to input, select, and textarea blocks in Create and Edit cshtml pages (similar to the blazor crud scenario). 
- above change fixes the narrator announcing "required" when above fields are selected

@MackinnonBuck need some feedback here pls.

### CODE REVIEW

File: eng/Versions.DotNetScaffold.props [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <VersionPrefix>9.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>10</PreReleaseVersionIteration>
+    <VersionPrefix>10.0.0</VersionPrefix>
```
Comment (tlmii) [inline eng/Versions.DotNetScaffold.props:4]: We should probably discuss with the team whether we want to tie vNext of dotnet-scaffold to v10 yet, or if an interim version is more appropriate for what we are targeting, before we ship a v10 preview to one of the feeds. Not the end of the world if we do ship it but doesn't hurt to have the conversation first I think.

File: src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap5/Edit.cshtml [hunk 2]
```diff
@@ -47,7 +47,8 @@
             @:<div asp-validation-summary="ModelOnly" class="text-danger"></div>
         foreach (PropertyMetadata property in Model.ModelMetadata.Properties)
         {
-            var propertyRequiredText = property.IsRequired ? "<span class=\"text-danger\">*</span>" : string.Empty;
+            var propertyRequiredText = property.IsRequired ? "\r\n<span class=\"text-danger\">*</span>" : string.Empty;
+            var ariaRequiredAttributeHtml = property.IsRequired ? "required aria-required=\"true\"" : string.Empty;
```
Comment (MackinnonBuck) [inline src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap5/Edit.cshtml:51]: Can we get away with removing `required` and just doing `aria-required`? The `required` attribute interferes with custom form validation.

Comment (MackinnonBuck) [reply src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap5/Edit.cshtml:51]: (same suggestion for other similar changes in this PR)

File: eng/Versions.DotNetScaffold.props (hunk 1 above)
Comment (deepchoudhery) [reply eng/Versions.DotNetScaffold.props:4]: sure yes can switch this back for now, will check how the darc feeds are flowing.

File: src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap5/Edit.cshtml (hunk 2 above)
Comment (deepchoudhery) [reply src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap5/Edit.cshtml:51]: will try in a sec here

Comment (deepchoudhery) [reply src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap5/Edit.cshtml:51]: yep it works without the `required`, removed

**Review by MackinnonBuck — APPROVED** [review]

**Pull request merged**